org.ekstep.questionset-1.0/renderer/utils/telemetry_logger.js

```diff
@@ -74,7 +74,8 @@ QSTelemetryLogger.logAssessEnd = function(result) {
   var data = {
     pass: result.eval,
     score: quesScore,
-    res: result.values,
+    params: result.params,
+    res: result.resValues,
```

Comment:
Why are changing the key name 
res: result.values => res: result.resValues

Keep the same so that old plugins of MCQ, FTB will still calling this method. If you change, it will fail for old published contents.